.github/workflows/CI.yaml

```diff
@@ -117,3 +117,11 @@ jobs:
           ./deployment/deploy_to_preview.sh dataland-backend/build/libs/dataland-backend*.jar
         env:
           SSH_PRIVATE_KEY: ${{ secrets.SSH_PRIVATE_KEY }}
+  summary:
+    runs-on: ubuntu-latest
+    if: success()
+    needs:
+      - sonar
+    steps:
+      - name: Summary of this workflows overall outcome
```

Comment:
Typo
Is: this workflows overall
Should: this workflow's overall

Reply:
fixed